packages/client/src/interceptors.ts

```diff
@@ -25,6 +25,22 @@ export interface WorkflowStartInput {
   readonly options: CompiledWorkflowOptions;
 }
 
+/** Input for WorkflowClientInterceptor.update */
+export interface WorkflowUpdateInput {
+  readonly updateName: string;
+  readonly args: unknown[];
+  readonly workflowExecution: WorkflowExecution;
+  readonly firstExecutionRunId?: string;
+  readonly headers: Headers;
+  readonly options: WorkflowUpdateOptions;
+}
+
+/** Output for WorkflowClientInterceptor.update */
+export interface WorkflowUpdateOutput {
+  readonly updateId: string;
+  readonly outcome?: temporal.api.update.v1.IOutcome | null;
```

Comment:
```suggestion
  readonly outcome?: temporal.api.update.v1.IOutcome | undefined;
```

Reply:
You may need to update the place that create that object to do `somevalue ?? undefined`. Still, I prefer that we explicitly use `null` in non-generated code.